src/brownie-in-mug.md

```diff
@@ -0,0 +1,36 @@
+# Brownie in a mug
+
+Simple recipe for a brownie microwaved and consumed in a mug.
+
+- ⏲️ Prep time: ~4 min
+- 🍳 Cook time: 1 min 40 seconds
+- 🍽️ Servings: 1
+
+## Ingredients
+
+- 4 tbsp (60ml) sugar
+- 4 tbsp (60ml) flour
+- 2 tbsp (30ml) cocoa
+- A pinch of salt
+- 2 tbsp (30ml) vegetable oil
+- 3 tbsp (45ml) water
+
+**(Optional)**
+- Milk
+- Vanilla sugar
+- Banana
+
+## Directions
+
+1. Add the dry ingredients to a regular sized coffee mug, preferably in the order they are listed in (easier to mix).
+2. Add first the oil, and then the water.
+3. Mix ingredients to a smooth paste with a tea spoon. Use the measurement cup, or another tea spoon, to put back any ingredients that stick to the spoon.
+4. Put the cup in the microwave for 1 minute 40 seconds.
+5. (Optional) Top it off with some vanilla sugar, and a sliced banana. Pour on some milk to cool it down.
+
+## Contribution
+
+- Anonymous - Creating a picture with the necessary ingredients and cooking time. Picture circulated on reddit, imgur, pintrest, among others.
+- raxiam - Compiling the recipe, and adding clear instructions.
+
+;tags: dessert snack basic baking
```

Comment:
```suggestion
;tags: dessert snack basic
```
Sorry, didn't catch this. 'Baking' is not a good tag, as most recipes involve baking, and the existing baking tag only has  a couple recipes, and should be removed soon. 
Also this recipe doesn't involve baking lol